src/components/users.vue

```diff
@@ -1,135 +1,142 @@
+              <td v-if="user">{{ user.role }}</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
     </div>
-    <table class="table table-bordered table-striped">
-      <thead>
-        <tr>
-          <sortable-th
-            :column="column"
-            :key="column"
-            :sort-ascending="sortAscending"
-            :sort-column="sortColumn"
-            @sort="sortBy"
-            class="sortable-th"
-            v-for="column in sortColumns"
-          ></sortable-th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr v-for="user in sortedUsers" :key="user.id">
-          <th>{{ user.id }}</th>
-          <td>{{ user.name }}</td>
-          <td>{{ user.email }}</td>
-          <td>
-            <span
-              v-for="application in user.applications"
-              :key="application.id"
-            >
-              {{ application.type }}
-            </span>
-          </td>
-          <td>{{ getMostRecentApplicationDate(user) }}</td>
-        </tr>
-      </tbody>
-    </table>
   </div>
 </template>
 
 <script>
+  import localforage from '../sessionUtils'
   import axios from 'axios';
   import sortableTh from '@/components/sortableTh';
+  import _ from 'lodash';
 
   const defaultSortAscending = true;
   const sortColumns = ['id', 'name', 'email', 'roles', 'date']
   const roles = ['admin', 'parent', 'sponsor', 'volunteer'];
 
   export default {
-    name: 'users',
-    components: {
-      'sortable-th': sortableTh,
-    },
-    data: function() {
+    data(){
       return {
-        filterRole: "",
-        roles: roles,
-        sortAscending: defaultSortAscending,
-        sortColumn: "id",
-        sortColumns: sortColumns,
+        filterString: '',
+        filteredUsers: [],
         users: [],
+        columns: ['name', 'email', 'registered', 'role'],
+        status: {
+          Name: 'fa fa-angle-right',
+          Email: 'fa fa-angle-right',
+          Registered: 'fa fa-angle-down',
+          Role: 'fa fa-angle-right'
+        }
       }
     },
     computed: {
-      sortedUsers: function() {
-        return this.filteredUsers.sort((left, right) => {
-          let leftColumn = null;
-          let rightColumn = null;
-          switch (this.sortColumn) {
-            case 'date':
-              leftColumn = this.getMostRecentApplicationDate(left);
-              rightColumn = this.getMostRecentApplicationDate(right);
-              break;
-            case 'roles':
-              leftColumn = left.applications.length;
-              rightColumn = right.applications.length;
-              break;
-            default:
-              leftColumn = left[this.sortColumn];
-              rightColumn = right[this.sortColumn];
+      filterUsers: function() {
+        this.filteredUsers = this.users.map(u => {
+          let date =  new Date(parseInt(u._id.$oid.substring(0, 8), 16) * 1000);
+          let shortDate = '' + date
```

Comment:
This is a minor nitpick, but I prefer explicit type conversion to type coercion. I think `let shortDate = date.toString()` would make it more clear what is actually happening on this line.

See Date.toString() for details: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toString